### DISCUSSION TITLE
Il mio sistema è in stato di inconsistenza a causa di mia operatività, è possibile resettare le segnalazioni di prova che ho fatto??

### DISCUSSION THREAD

**Discussion by paolo987** [globaleaks/globaleaks-whistleblowing-software#3881] [Q&A] [answered]
data la discussion #3879 e #3827 
credo che la cancellazione delle chiavi degli utenti che ho fatto da DB abbia generato dei problemi sulle segnalazioni precedenti che avevo fatto (tutte di prova).

accedendo come "case manager" ottengo un errore interno del server se faccio click su "Segnalazioni"
![image](https://github.com/globaleaks/GlobaLeaks/assets/78037015/57082905-e1b6-4bca-9a25-720234558ede)

invece se dalla mail che riceve il case manager, un nuovo case faccio click sul link vedo il case posso gestirlo... per poi ottenere lo stesso errore se faccio click sull'icona "Segnalazioni"

e ricevo questa mail:
`Platform: Whistleblowing XXXX Srl
Host: XXXX (XXXXX)
Version: 4.13.21

nacl.exceptions.CryptoError 
    Base exception for all nacl related errors
    

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/twisted/python/threadpool.py", line 244, in inContext
    result = inContext.theWork()  # type: ignore[attr-defined]
             ^^^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3/dist-packages/twisted/python/threadpool.py", line 260, in <lambda>
    inContext.theWork = lambda: context.call(  # type: ignore[attr-defined]
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3/dist-packages/twisted/python/context.py", line 117, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3/dist-packages/twisted/python/context.py", line 82, in callWithContext
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3/dist-packages/globaleaks/orm.py", line 185, in _wrap
    result = function(session, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3/dist-packages/globaleaks/handlers/recipient/__init__.py", line 79, in get_receivertips
    tip_key = GCE.asymmetric_decrypt(user_key, base64.b64decode(rtip.crypto_tip_prv_key))
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3/dist-packages/globaleaks/utils/crypto.py", line 305, in asymmetric_decrypt
    return SealedBox(prv_key).decrypt(data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3/dist-packages/nacl/public.py", line 417, in decrypt
    plaintext = nacl.bindings.crypto_box_seal_open(
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/lib/python3/dist-packages/nacl/bindings/crypto_box.py", line 318, in crypto_box_seal_open
    ensure(

  File "/usr/lib/python3/dist-packages/nacl/exceptions.py", line 88, in ensure
    raise raising(*args)

nacl.exceptions.CryptoError: An error occurred trying to decrypt the message
`

è possibile cancellare tutte le segnalazioni presenti su DB ma mantenere i settings che ci sono?
Ammesso che il problema possa essere questo.

**Comment by paolo987** [comment]
up

**Comment by evilaliv3** [comment, marked as answer]
Gentile @paolo987 

Esiste la possibilita di cancellare le segnalazioni che trova in fondo alle imostazioni avanzate, è un bottone rosso "Resetta segnalazioni".

La preghiamo di portare maggiore pazienza nelle risposte. Ogni ticket viene da noi esaminato da noi e supportato in ottica di collaborazione e secondo priorità cercando di supportare volontariamente tutta la community. La ringraziamo per la comprensione.

**Reply by paolo987** [reply]
Grazie del supporto @evilaliv3 
Resettando le segnalazioni il problema è rientrato.
Perdonami ma mi serviva risposta entro la giornata in modo da essere operativo al 100%. Ora lo sono.
Ancora grazie e complimenti per la soluzione realizzata.
